openquake/hazardlib/gsim/kuehn_2020.py

```diff
@@ -25,14 +25,16 @@
                :class:`KuehnEtAl2020SInterNewZealand`,
                :class:`KuehnEtAl2020SInterSouthAmerica`,
                :class:`KuehnEtAl2020SInterTaiwan`,
+               :class:`KuehnEtAl2020SInterSeattle`,
```

Comment:
The name is confusing since it mixes with the regional adjustments (Does 'KuehnEtAl2020SInterCascadiaSeattleBasin' sound correct?).

Reply:
I think your suggestion makes more sense. I've updated the classes/aliasses to use your suggestion instead.